src/devtools/views/actionList/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import css from './actionList.css'
+import Item from './item'
+
+class ActionList extends React.Component {
+  render() {
+    let { list, head } = this.props.history
+
+    return (
+      <div className={css.container}>
+        {list.map(action => (
+          <Item action={action} head={head} key={action.id} />
+        ))}
+      </div>
```

Comment:
I like to do this as:

```
renderItem (action) {
    return <Item action={action} head={head} key={action.id} />
}

render () {
  // ...
  list.map(this.renderItem, this)
}
```

The second argument of `map` is scope, so it keeps it to that of this component class. This is also totally style. What do you like better?